src/frequenz/sdk/actor/_run_utils.py

```diff
@@ -19,8 +19,12 @@ async def run(*actors: Actor) -> None:
         actors: the actors to be awaited.
     """
     _logger.info("Starting %s actor(s)...", len(actors))
+
+    for actor in actors:
+        actor.start()
```

Comment:
The log message about actors starting should be moved here. I was thinking it might be nice to also have an `if actor.is_running` and print a different message if the actor was already started than if we just started it, maybe it could be helpful when debugging if one didn't realize that one actor was started before the `run()` or something.